### ISSUE TITLE
Make sure that it can load perfectly from a 1.7.10 buildcraft modded world

### ISSUE THREAD

**Issue by AlexIIL** [AlexIIL/Buildcraft_1.8_Issues#30]
And I mean everything dammit, not just some parts, everything must load properly.
- [ ] The quarry doesn't migrate
- [ ] Most blocks have strange rotations
- [ ] The chute doesn't migrate (But it is deprecated anyway so... what to do?)
- [ ] Fuel and oil doesn't migrate

**Comment by AlexIIL** [comment]
STATUS UPDATE:

EVERYTHING WORKS EXCEPT FOR WHAT IS IN THE DESC

WHY AM I SHOUTING

SHOUTING WORDS OF JOY, THATS WHY

_ahem_

**Comment by AEnterprise** [comment]
for the chute i'd say also migrate it, while depreciated still used or someone could still have one in an old build that would break otherwise

**Comment by AlexIIL** [comment]
I was referring that if I couldn’t get the chute to migrate (for WHATEVER reason) then I would be ok with it. If I can, it will be migrated.

**Comment by Xariyl** [comment]
Agreed. I've come to like the chute quite a bit, so if it's not too much of a hassle, I'd also recommend/request including it. 

It's a bit cheaper than the hopper, faster, doesn't play funny if redstone is next to it, can be extracted from using a RS Engine without the need of a wooden pipe... as it no longer needs iron gears it is a viable alternative if you don't need the "advanced" features of the hopper. 

I wouldn't cry if there were a few more slots available... Just saying :)

**Comment by AlexIIL** [comment]
So it turns out the chute is migrating. However the quarry is being stubborn. No-one uses that piece of junk so we might as well forget about it

_(Heavy sarcasm detected)_
